apps/air-discount-scheme/api/src/app/app.module.ts

```diff
@@ -10,15 +10,14 @@ import {
   FlightLegModule,
 } from './modules'
 import { BackendAPI } from '../services'
+import { environment } from '../environments'
 
 const debug = process.env.NODE_ENV === 'development'
 const playground = debug || process.env.GQL_PLAYGROUND_ENABLED === 'true'
-const autoSchemaFile = debug ? 'apps/air-discount-scheme/api.graphql' : true
+const autoSchemaFile = environment.production
```

Comment:
Weird to have a variable that either has a boolean or a string

Reply:
Well, that's a decision made by the creators of `@nestjs/graphql`. I agree, it's super confusing. `true` means in-memory btw.

Reply:
Then they are weird